src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -286,22 +286,22 @@ def to_proto(self):
     crash.file_path = self.file_path
     crash.crash_time = self.crash_time
     crash.return_code = self.return_code
-    crash.resource_list = self.resource_list
-    crash.gestures = self.gestures
+    crash.resource_list.extend(self.resource_list)
+    crash.gestures.extend(self.gestures)
```

Comment:
I had unit tests break for the 3.11 bump pr, is this the issue you are tackling?

```
======================================================================
ERROR: test_create_many_groups_0_some_project (tests.core.bot.tasks.utasks.fuzz_task_test.with_cloud_emulators.<locals>.decorator.<locals>.Wrapped.test_create_many_groups_0_some_project)
Test creating many groups [with project_name='some_project'].
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/workspace/.venv/lib/python3.11/site-packages/parameterized/parameterized.py", line 620, in standalone_func
    return func(*(a + p.args), **p.kwargs, **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/fuzz_task_test.py", line 815, in test_create_many_groups
    groups = fuzz_task.process_crashes(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py", line 1126, in process_crashes
    main_crash=group.main_crash.to_proto(),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py", line 289, in to_proto
    crash.resource_list = self.resource_list
    ^^^^^^^^^^^^^^^^^^^
AttributeError: Assignment not allowed to message, map, or repeated field "resource_list" in protocol message object.

======================================================================
ERROR: test_create_many_groups_1_chromium (tests.core.bot.tasks.utasks.fuzz_task_test.with_cloud_emulators.<locals>.decorator.<locals>.Wrapped.test_create_many_groups_1_chromium)
Test creating many groups [with project_name='chromium'].
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/workspace/.venv/lib/python3.11/site-packages/parameterized/parameterized.py", line 620, in standalone_func
    return func(*(a + p.args), **p.kwargs, **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/fuzz_task_test.py", line 815, in test_create_many_groups
    groups = fuzz_task.process_crashes(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py", line 1126, in process_crashes
    main_crash=group.main_crash.to_proto(),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py", line 289, in to_proto
    crash.resource_list = self.resource_list
    ^^^^^^^^^^^^^^^^^^^
AttributeError: Assignment not allowed to message, map, or repeated field "resource_list" in protocol message object.

======================================================================
ERROR: test_existing_unreproducible_testcase (tests.core.bot.tasks.utasks.fuzz_task_test.with_cloud_emulators.<locals>.decorator.<locals>.Wrapped.test_existing_unreproducible_testcase)
Test existing unreproducible testcase.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/workspace/src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/fuzz_task_test.py", line 742, in test_existing_unreproducible_testcase
    groups = fuzz_task.process_crashes(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py", line 1126, in process_crashes
    main_crash=group.main_crash.to_proto(),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py", line 289, in to_proto
    crash.resource_list = self.resource_list
    ^^^^^^^^^^^^^^^^^^^
AttributeError: Assignment not allowed to message, map, or repeated field "resource_list" in protocol message object.
```

Reply:
I am fixing it yup. Sorry!

Reply:
Latest push should fix it.